src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -107,59 +109,76 @@ export default class Result extends React.Component {
 
     return (
       <div>
-        {this.renderToggle()}
+        <Flex align="center" mb={space(2)}>
+          <Box flex="1">
+            <Heading>Result</Heading>
+          </Box>
+          <Box justifySelf="flex-end">{this.renderToggle()}</Box>
+        </Flex>
 
         {view === 'table' && <Table data={data} query={query} />}
         {view === 'line' && (
-          <LineChart
-            series={basicChartData}
-            height={300}
-            options={{
-              tooltip: Tooltip({
-                formatter: formatTooltip,
-              }),
-            }}
-          />
+          <Panel>
+            <Box px={space(2)} py={space(1.5)}>
```

Comment:
Can we change this into a styled component? It's re-used multiple times here and also we should limit the use of these presentational props -- it makes things hard to read.

Reply:
👍